src/EnergyPlus/HybridEvapCoolingModel.cc

```diff
@@ -0,0 +1,1591 @@
+
+#include <HybridEvapCoolingModel.hh>
+
+#include <UtilityRoutines.hh>
+
+#include <string>
+#include <list>
+#include <math.h>
+#include <windows.h>
+#include <ScheduleManager.hh>
+#include <General.hh>
+#include <CurveManager.hh>
+#include <DataGlobals.hh>
+#include <DataGlobalConstants.hh>
+#include <DataHVACGlobals.hh>
+#include <Psychrometrics.hh>
+#include <DataEnvironment.hh>
+#include <DataZoneEquipment.hh>
+// ObjexxFCL Headers
+#include <ObjexxFCL/Array.functions.hh>
+#include <ObjexxFCL/Fmath.hh>
+using namespace std;
+
+namespace EnergyPlus {//***************
+
+	namespace HybridEvapCoolingModel {
+		using namespace std;
+		using DataHVACGlobals::SysTimeElapsed;
+		using DataHVACGlobals::NumOfSysTimeSteps;
+		using DataHVACGlobals::NumOfSysTimeStepsLastZoneTimeStep;
+		using DataHVACGlobals::LimitNumSysSteps;
+		using DataHVACGlobals::TimeStepSys;
+		using DataGlobals::SecInHour;
+		using namespace EnergyPlus::DataEnvironment;
+		using namespace Psychrometrics;
+		using CurveManager::GetCurveIndex;
+		using CurveManager::GetCurveType;
+		using CurveManager::GetCurveMinMaxValues;
+		using CurveManager::CurveValue;
+		using EnergyPlus::CurveManager::GetNormalPoint;
+		using ScheduleManager::GetCurrentScheduleValue;
+		typedef int* MyType;
+		map<string, string> testop;
+#define  DEF_Tdb 0
+#define  DEF_RH 1
+
+#define  TEMP_CURVE 0 
+#define  W_CURVE 1
+#define  POWER_CURVE 2
+		// constructor of Model,
+
+#include <fstream>
+		fstream logfile;
+		//ZoneHybridUnitaryACSystem
+
+		bool CMode::InitializeOutsideAirTemperatureConstraints(double min, double max)
```

Comment:
It's weird having all these init routines that do essentially the same thing, assigning two arguments to two variables.  They have comments that are different, but ultimately the are exactly the same function.  I could understand having all these split out into meaningful functions if they were actually defaulting to specific things.  For example:

```
bool CMode::InitializeOutsideAirTemperatureConstraints()
{
  Min_OA_Temp = 0.0;
  Max_OA_Temp = 100;
}
```

But since they are require you to actually pass in meaningful values each time, I don't see how it's better than just initializing the variables where you need to initialize them.  Instead of:

```
InitializeOutsideAirTemperatureConstraints(0.0, 100.0);
```

do:

```
  Min_OA_Temp = 0.0;
  Max_OA_Temp = 100;
```